testutil/ciao-down/workloads/clearcontainers-3.x.yaml

```diff
@@ -0,0 +1,249 @@
+{{ define "GOPATH" }}{{with .GoPath}}{{$.MountPath "hostgo"}}{{else}}/home/{{.User}}/go{{end}}{{end}}
+#cloud-config
+write_files:
+{{- if len $.HTTPProxy }}
+ - content: |
+     [Service]
+     Environment="HTTP_PROXY={{$.HTTPProxy}}"{{if len .HTTPSProxy}} "HTTPS_PROXY={{.HTTPSProxy}}{{end}}"{{if len .NoProxy}} "NO_PROXY={{.NoProxy}},{{.Hostname}}{{end}}"
+   path: /etc/systemd/system/docker.service.d/http-proxy.conf
+{{- end}}
+{{with proxyEnv . 5}}
+ - content: |
+{{.}}
+   path: /etc/environment
+{{end}}
+ - content: |
+     [Service]
+     ExecStart=
+     ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/bin/cc-runtime --default-runtime=cor
+   path: /etc/systemd/system/docker.service.d/clr-containers.conf
+ - content: |
+     #!/bin/sh
+     printf "\n"
+     printf "\n"
+     printf "Your go code is at {{template "GOPATH" .}}\n"
+     printf "You can also edit your code on your host system \n"
+     printf "To build cc-runtime from sources \n"
+     printf "go get -d github.com/clearcontainers/runtime... \n"
+     printf "cd {{template "GOPATH" .}}/src/github.com/clearcontainers/runtime \n"
+     printf "make build-cc-system\n"
+     printf "sudo -E PATH=$PATH make install-cc-system \n"
+     printf "\n"
+     printf "\n"
+   path: /etc/update-motd.d/10-ciao-help-text
+   permissions: '0755'
+ - content: |
+     [Unit]
+      Description=OCI-based implementation of Kubernetes Container Runtime Interface
+      Documentation=https://github.com/kubernetes-incubator/cri-o
+
+      [Service]
+      ExecStart=/usr/bin/crio --debug
+      Environment="HTTP_PROXY={{$.HTTPProxy}}"{{if len .HTTPSProxy}} "HTTPS_PROXY={{.HTTPSProxy}}{{end}}"{{if len .NoProxy}} "NO_PROXY={{.NoProxy}},{{.Hostname}}{{end}}"
+      Restart=on-failure
+      RestartSec=5
+
+      [Install]
+      WantedBy=multi-user.target
+   path: /etc/systemd/system/crio.service
+ - content: |
+     deb https://apt.dockerproject.org/repo ubuntu-xenial main
+   path: /etc/apt/sources.list.d/docker.list
+
+apt:
+{{- if len $.HTTPProxy }}
+  proxy: "{{$.HTTPProxy}}"
+{{- end}}
+{{- if len $.HTTPSProxy }}
+  https_proxy: "{{$.HTTPSProxy}}"
+{{- end}}
+package_upgrade: {{with .PackageUpgrade}}{{.}}{{else}}true{{end}}
+
+runcmd:
+ - {{beginTask . "Booting VM"}}
+ - {{endTaskOk . }}
+
+ - {{beginTask . (printf "Adding %s to /etc/hosts" .Hostname) }}
+ - echo "127.0.0.1 {{.Hostname}}" >> /etc/hosts
+ - {{endTaskCheck .}}
+
+{{range .Mounts}}
+ - mkdir -p {{.Path}}
+ - sudo chown {{$.User}}:{{$.User}} {{.Tag}}
+ - echo "{{.Tag}} {{.Path}} 9p x-systemd.automount,x-systemd.device-timeout=10,nofail,trans=virtio,version=9p2000.L 0 0" >> /etc/fstab
+{{end}}
+{{range .Mounts}}
+ - {{beginTask $ (printf "Mounting %s" .Path) }}
+ - mount {{.Path}}
+ - {{endTaskCheck $}}
+{{end}}
+
+ - chown {{.User}}:{{.User}} /home/{{.User}}
+ - rm /etc/update-motd.d/10-help-text /etc/update-motd.d/51-cloudguest
+ - rm /etc/update-motd.d/90-updates-available
+ - rm /etc/legal
+
+ - {{beginTask . "Booting VM"}}
```

Comment:
This message is already printed out right at the top.  Is there any reason to print it again?

Reply:
nope -- this was either a copy paste issue on my side, or an error in the original.  Will remove.

Reply:
And, turns out it was this way in original as well - I'll fix both.